docs/usage/11-provisioning-tcp-cloud-loadbalancers.adoc

```diff
@@ -28,70 +28,46 @@ As of this writing, only [AWS ELB](https://aws.amazon.com/elasticloadbalancing/)
 
 === Specifying the configuration for the cloud load-balancer
 
-`dklb` must read the cloud load-balancer's configuration from a https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/[config map].
-This config map must exist in the same Kubernetes namespace as the `Service` resource being exposed via a cloud load-balancer, and must obey the following structure:
-
-[source,yaml]
-----
-apiVersion: v1
-data:
-  spec: |
-    <spec>
-kind: ConfigMap
-metadata:
-  name: "<cloud-loadbalancer-configmap>"
-----
-
-In particular, it must have a key named `spec` containing the desired configuration for the cloud load-balancer.
-Further details on `<spec>` are provided below.
-
-=== Linking the configuration to the `Service` resource
-
-In order for `dklb` to provision a cloud load-balancer for a given `Service` resource, said resource must be annotated with the following annotation:
+`dklb` must read the _raw_, JSON-encoded cloud load-balancer's configuration from the `.cloudProviderConfiguration` field of the configuration object.
```

Comment:
Isn't there yet an official release to this feature? [Asked on slack too](https://mesosphere.slack.com/archives/GETU01B88/p1554803163000600).

Reply:
I don't believe there is, especially because `aws-automation` hasn't been merged yet.

Reply:
That's unfortunate as we're way past the GA release date 🤔 right @chrisgaun?